packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/Domain/ASyncImageProcessor.cs

```diff
@@ -200,10 +200,11 @@ public class ASyncImageProcessor : IHostedService, IDisposable {
             if (!Directory.Exists(filepath)) {
                 _logger.LogInformation("Creating directory for images " + filepath);
                 Directory.CreateDirectory(filepath);
-            }
-
-            Thread.Sleep(3000);
 
+                // wait for new folder to be writable
+                Thread.Sleep(3000);

```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.createdirectory does not imply the creation of a directory is an async operation. Also the examples from MS suggests that it is a blocking operation. We should therefore not require sleeping for 3 seconds.

Reply:
Agreed and so i thought (note that link is not the .net core SDK). 
However I encountered situations where directory does not exist exceptions were generated without this. It is possible that it may have been an interaction of docker, libvips and my install 
so i am willing to remove it but note it may introduce an instability.

Reply:
Why not catching the exception and retry it X times (or for a time of Y seconds)?
Or wait until the folder is writable with polling in a loop? Something like:  `while(!Directory.Writable && retries < X){}`

This way we are not putting a hard threshold to the performance of the code.

Reply:
@miguems i didn't take the approach of catching and retrying the exception as it was thrown not while interacting with .net code but while interacting with libvips which reported the output location unwritable under docker. 

As i say this might have been a one-off or specific to my docker install or an earlier and now outdated docker script.  

I will seek to remove this in a future PR after testing. 

However i think this is outside of the scope of this PR.